lib/Google/RestApi.pm

```diff
@@ -21,6 +21,8 @@ use URI;
 use URI::QueryParam;
 use YAML::Any qw(Dump LoadFile);
 
+use WWW::Google::Cloud::Auth::ServiceAccount;
```

Comment:
Is there any need to make this stand out on its own? I'd put it above YAML::Any and remove the extra whitespace.

Reply:
no, there is not. just did not want to intrude existing structure